website/src/pages/repl.js

```diff
@@ -1,22 +0,0 @@
-import react from "react";
-import Layout from "@theme/Layout";
-
-export default function Repl() {
-  react.useEffect(() => {
-    const script = document.createElement("script");
-    script.src = "/repl/index.js";
-    document.body.appendChild(script);
-    return () => {
-      document.body.removeChild(script);
-    };
-  }, []);
-  return (
-    <div>
-      <Layout noFooter={true}>
```

Comment:
The `noFooter={true}` should be added to the `repl.tsx`

Reply:
The Footer is visible if one scrolls to the bottom from the left toolbar, hence we used the `noFooter` property.

<img width="1055" alt="image" src="https://github.com/user-attachments/assets/2322e4a7-4310-4310-a17b-2a04c2707729" />